docs/getting-started/pull-request.md

```diff
@@ -6,20 +6,20 @@ title: Preparing a Pull Request
 
```

Comment:
this was copied and pasted from the contributing guide section, let's not keep these two documents unaligned